app/test/account/like_test.dart

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) 2019, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:test/test.dart';
+
+import '../shared/test_models.dart';
+import '../shared/test_services.dart';
+
+void main() {
+  group('/api/account/likes', () {
+    testWithServices('Like package', () async {
+      final client = createPubApiClient(authToken: hansUser.userId);
+      final rs = await client.getLikePackage('hydrogen');
+      expect(rs.package, 'hydrogen');
+      expect(rs.liked, false);
+
+      final rs2 = await client.likePackage('hydrogen');
+      expect(rs2.package, 'hydrogen');
+      expect(rs2.liked, true);
+
+      final rs3 = await client.getLikePackage('hydrogen');
+      expect(rs3.package, 'hydrogen');
+      expect(rs3.liked, true);
```

Comment:
Lovely tests :)

Can we try to like the package again... to see that this operation has no impact and doesn't cause an exception when the package is already liked.